### PR TITLE
flaky test test_bkp fixes

### PR DESCRIPTION
**Pull request by vivek-infracloud**
Logic breaks at cmd_output, because ping command cannot reach to the pod.
So, added a retry ping logic to check again if the pod is reachable.

### CODE REVIEW

File: tests/validation/tests/v3_api/common.py [hunk 1]
```diff
@@ -1026,6 +1026,13 @@ def validate_dns_entry(pod, host, expected, port=TEST_IMAGE_PORT):
         cmd = 'ping -c 1 -W 1 {0}'.format(host)
     cmd_output = kubectl_pod_exec(pod, cmd)
 
+    for i in range(0, retry_count):
+        cmd_output = kubectl_pod_exec(pod, cmd)
+        if str(pod.name) in str(cmd_output):
```
Comment (izaac) [inline tests/validation/tests/v3_api/common.py:1031]: Could we add this validation before entering the `for` loop so we don't retry on an already successful output.

**Review by izaac — CHANGES_REQUESTED** [review]

File: tests/validation/tests/v3_api/common.py (hunk 1 above)
Comment (vivek-infracloud) [reply tests/validation/tests/v3_api/common.py]: I've added a logic. Please review again.

**Comment by izaac** [comment]
Please squash the commits

**Review by izaac — APPROVED** [review]
LGTM

**Review by anupama2501 — APPROVED** [review]
lgtm

**Comment by vivek-infracloud** [comment]
@sowmyav27 @snasovich please review the PR.

**Review by snasovich — APPROVED** [review]

**Pull request merged**